### ISSUE TITLE
Should we pay attention to Warning: no ambient RNA beads were found; maybe sample had too few cells?

### ISSUE THREAD

**Issue by denvercal1234GitHub** [linnarsson-lab/loompy#162]
Hi there, 

I was able to generated loom file from my fastq ran by 10X, but looking at the .out log, there was a warning "no ambient RNA beads were found; maybe sample had too few cells?"

Should we worry or do something about this? And, what would be the impact of this in the downstream analysis or interpretation of the results? 

Thank you so much for your help!

**Comment by AY-LIANG** [comment]
Hi,
Have you solved this problem?I am experiencing the same warning,and I get only few cells in the result.

**Comment by aelhossiny** [comment]
Same thing, getting 29 cells only!

**Comment by pranzatelli** [comment]
I agree with both of these - high quality scRNA datasets are returning double-digit numbers of cells. I think that #127 might be a relevant (also open) issue. Until this is fixed, loompy is unusable for these projects.
